Sources/ZLImageEditor.bundle/de.lproj/Localizable.strings

```diff
@@ -1,9 +1,9 @@
-"cancel" = "Stornieren";
+"cancel" = "Abbrechen";
```

Comment:
Thanks for your pr. But I want to mean cancel, not abort. I see the text in WeChat is also 'Stornieren'.
<img width="769" alt="image" src="https://user-images.githubusercontent.com/11711768/195011877-0138d534-ed34-40d2-8d95-787dd51d9e0c.png">

Reply:
Nobody in Germany uses "Stornieren". And WeChats translation is totally off anyway. As a native speaker, I can tell you that "Abbrechen" is the correct term here.

Reply:
We use "Stornieren" only when we have like a running Payment or Shipping and wan't to cancel this out, but not in this context for a button.

Reply:
OK~